spectral_cube/spectral_cube.py

```diff
@@ -3197,7 +3208,11 @@ def convfunc(img, **kwargs):
                             **kwargs)
 
         newcube = self.apply_function_parallel_spatial(convfunc,
-                                                       **kwargs).with_beam(beam)
+                                                       **kwargs).with_beam(beam, raise_error_jybm=False)
+
+        # Scale Jy/beam units by the change in beam size
+        if self.unit.is_equivalent(u.Jy / u.beam):
+            newcube *= (beam.sr / self.beam.sr).value
```

Comment:
is there any way we could include this scaling in the convolution to avoid an additional pass over the data?  I think not, because of the `normalize_kernel` step, but maybe we could change `_gsmooth_image` from `convolve(img,` to `convolve(img*scalefactor,`

Reply:
I've fixed this and included it in the convolve function (like the other 3 cases had). All 4 convolve functions now do the correction per channel.